cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -303,17 +318,23 @@ spec:
         # Rook Agent toleration. Will tolerate all taints with all keys.
         # Choose between NoSchedule, PreferNoSchedule and NoExecute:
         # - name: AGENT_TOLERATION
-        #  value: "NoSchedule"
+        #   value: "NoSchedule"
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
-        #  value: "<KeyOfTheTaintToTolerate>"
+        #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent mount security mode. Can by `Any` or `Restricted`.
+        # `Any` uses Ceph admin credentials by default/fallback.
+        # For using `Restricted` you must have a Ceph secret in each namespace storage should be consumed from and
+        # set `mountUser` to the Ceph user and `mountSecret` to the Kubernetes secret name with the Ceph user key.
+        # - name: AGENT_MOUNT_SECURITY_MODE
```

Comment:
Why do we need this flag? The storage class settings aren't sufficient to trigger the behavior?

Reply:
It is to enforce the mount security mode that not `admin` is "accidentally" used.